specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2024-04-01-preview/fluxconfiguration.json

```diff
@@ -0,0 +1,1822 @@
+      "type": "object",
+      "x-nullable": true,
+      "properties": {
+        "substitute": {
+          "description": "Key/value pairs holding the variables to be substituted in this Kustomization.",
+          "type": "object",
+          "x-nullable": true,
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "substituteFrom": {
+          "description": "Array of ConfigMaps/Secrets from which the variables are substituted for this Kustomization.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/SubstituteFromDefinition"
+          },
+          "x-ms-identifiers": [],
+          "x-nullable": true
+        }
+      }
+    },
+    "PostBuildPatchDefinition": {
+      "description": "The postBuild definitions defining variable substitutions for this Kustomization after kustomize build.",
+      "type": "object",
+      "x-nullable": true,
+      "properties": {
+        "substitute": {
+          "description": "Key/value pairs holding the variables to be substituted in this Kustomization.",
+          "type": "object",
+          "x-nullable": true,
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "substituteFrom": {
+          "description": "Array of ConfigMaps/Secrets from which the variables are substituted for this Kustomization.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "$ref": "#/definitions/SubstituteFromPatchDefinition"
+          },
+          "x-ms-identifiers": [],
+          "x-nullable": true
+        }
+      }
+    },
+    "SubstituteFromDefinition": {
+      "description": "Array of ConfigMaps/Secrets from which the variables are substituted for this Kustomization.",
+      "type": "object",
+      "x-nullable": true,
+      "properties": {
+        "kind": {
+          "description": "Define whether it is ConfigMap or Secret that holds the variables to be used in substitution.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of the ConfigMap/Secret that holds the variables to be used in substitution.",
+          "type": "string"
+        },
+        "optional": {
+          "description": "Set to True to proceed without ConfigMap/Secret, if it is not present.",
+          "type": "boolean",
+          "default": false
+        }
+      }
+    },
+    "SubstituteFromPatchDefinition": {
+      "description": "Array of ConfigMaps/Secrets from which the variables are substituted for this Kustomization.",
+      "type": "object",
+      "x-nullable": true,
+      "properties": {
+        "kind": {
+          "description": "Define whether it is ConfigMap or Secret that holds the variables to be used in substitution.",
+          "type": "string"
+        },
+        "name": {
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

Property names within the property bag need to be distinct from top-level ARM property names (name, id, type, systemData, etc.). Consider configName or something else.

Reply:
this is already in stable version and not part of the changes that are been made in this preview version.